scripts/ci-checks.sh

```diff
@@ -25,6 +25,14 @@ else
   "$SCRIPT_DIR"/check-format.sh src samples
 fi
 
+npm --loglevel=error install --save-dev --save-exact prettier 1>/dev/null
```

Comment:
Why `--save-exact`?

Reply:
In fact, why save at all? Will this create a package.json and/or a node_modules in the repo root?

Reply:
Just following doc recommendations, and it does create those. I've removed those options.